multiqc/utils/search_patterns.yaml

```diff
@@ -82,6 +82,32 @@ biobloomtools:
     num_lines: 2
 bismark/align:
     fn: '*_[SP]E_report.txt'
+biscuit/mapq:
+    - fn: '*_mapq_table.txt'
+    - fn: '*_strand_table.txt'
+    - fn: '*_isize_score_table.txt'
+biscuit/markdup:
+    fn: '*_markdup_report.txt'
+biscuit/coverage:
+    - fn: '*_all_cv_table.txt'
+    - fn: '*_covdist_cpg_q40_table.txt'
+    - fn: '*_covdist_cpg_q40_botgc_table.txt'
+    - fn: '*_covdist_cpg_q40_topgc_table.txt'
+    - fn: '*_covdist_cpg_table.txt'
+    - fn: '*_covdist_q40_botgc_table.txt'
+    - fn: '*_covdist_q40_table.txt'
+    - fn: '*_covdist_q40_topgc_table.txt'
+    - fn: '*_covdist_table.txt'
+    - fn: '*_cpg_cv_table.txt'
+    - fn: '*_cpg_dist_table.txt'
+    - fn: '*_dup_report.txt'
```

Comment:
Same as above with the common filename pattern.

Reply:
Same solution as in response above. Thanks.